gen/topUtils.ml

```diff
@@ -281,7 +282,8 @@ module Make : functor (O:Config) -> functor (C:ArchRun.S) ->
         Some r
       with Not_found -> None
 
-    let is_load_init e = e.C.C.dir = Some R && e.C.C.v = 0
+    (* TODO check if the initial should be change to None? *)
```

Comment:
Notice that. by litmus test convention, locations that are not explicitly initialized are in fact initialized to zero.